.evergreen/.evg.yml

```diff
@@ -688,3 +705,10 @@ buildvariants:
   run_on: ubuntu1804-test
   tasks:
     - name: "publish-snapshot"
+
+# Storage engine tests on RHEL 6.2 (x86_64)
```

Comment:
Remove comment.  That's the only place we test anything!

Reply:
Ping on this

Reply:
Removed.